python/src/aac/plugins/gen_plugin/templates/pyproject_toml.jinja2

```diff
@@ -0,0 +1,76 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "{{project.name}}"
+version = "0.0.1"
+authors = [
+	{ name="AaC Dev Team", email="asdfasdaf@email.com" },
+]
+description = "{{ project.description }}"
+requires-python = ">= 3.9.13"
+readme = "README.md"
+license = {text = "MIT License"}
+keywords = ["SysML", "Architecture-as-Code", "System Engineering", "MBSE"]
+classifiers = [
+	"Programming Language :: Python :: 3",
+	"License :: OSI Approved :: MIT License",
+	"Operating System :: OS Independent",
+	"Topic :: Scientific/Engineering"
+]
+
+dependencies = [
+    "aac ~= {{ aac_version() }}",
+    {% for uses in project.uses %}"{{uses.name}} ~= {{uses.version}}",
+    {% endfor %}
+]
+[project.optional-dependencies]
+doc = [
+	"sphinx >= 7.2.6",
+	"sphinx-autobuild ~= 2021.3.14",
+	"sphinx_contributors >= 0.2.7",
+	"sphinx-copybutton >= 0.5.2",
+	"sphinxcontrib-applehelp ~= 1.0.2",
+    "sphinxcontrib-devhelp ~= 1.0.2",
+    "sphinxcontrib-htmlhelp ~= 2.0.0",
+    "sphinxcontrib-jsmath ~= 1.0.1",
+    "sphinxcontrib-qthelp ~= 1.0.3",
+    "sphinxcontrib-serializinghtml ~= 1.1.9",
+	"docutils ~= 0.19",
+	"furo >= 2024.1.29",
+	"myst-parser >= 2.0.0",
+	"pytz ~= 2023.3"
+]
+test = [
+	"behave >= 1.2.6",
+	"build >= 1.1.1",
+	"coverage >= 7.4.3",
+	"flake8 >= 4.0.0",
+	"nose2 >= 0.10.0",
+	"pipdeptree >= 2.16.1",
+	"pyflakes >= 2.4",
+	"tox >= 3.24",
+    "httpx ~= 0.27.0"
+]
+
+all = [
+	"aac >= 0.4.8",
+	"{{project.name}}[doc]",
+	"{{project.name}}[test]"
+]
+
+[project.entry-points."aac"]
+{{get_python_name(project.name)}} = "{{get_python_name(project.name)}}"
```

Comment:
I tried looking at the legacy template and in the other repos...So, legacy just took project name and didn't manipulate it any for the left side of the equals sign. And current issue is that while, yes, the definition allows spaces, will pass check, can be searched for, etc., we don't have any fielded with spaces to know if this will handle as we expect with spaces for registering the entry point. 

Sorry for being back and forth, but think we should go back to just `project.name` and worry about altering/using a helper method if we actually hit any issues. 

Thoughts?